spec/requests/providers/application_merits_task/is_matter_opposed_controller_spec.rb

```diff
@@ -0,0 +1,168 @@
+require "rails_helper"
+
+RSpec.describe Providers::ApplicationMeritsTask::IsMatterOpposedController do
+  let(:legal_aid_application) do
+    create(
+      :legal_aid_application,
+      :with_proceedings,
+      explicit_proceedings: %i[pbm40],
+    )
+  end
+
+  let(:provider) { legal_aid_application.provider }
+
+  describe "GET /providers/applications/:legal_aid_application_id/is_matter_opposed" do
+    subject(:request) do
+      get providers_legal_aid_application_is_matter_opposed_path(legal_aid_application)
+    end
+
+    it "returns ok" do
+      login_as provider
+
+      request
+
+      expect(response).to have_http_status(:ok)
+    end
+
+    context "when the application has a matter opposition" do
+      it "displays the reason" do
+        _matter_opposition = create(
```

Comment:
Do you need to assign this to a variable at all? Can you just do `create(...`?

Reply:
I can, I copied it from the _other_ matter opposed controller and Connor loved assigning variables, even non-used ones!